README.md

```diff
@@ -1,5 +1,25 @@
-Bare Metal Operator
-===================
+# Bare Metal Operator
+
+The Bare Metal Operator implements a Kubernetes API for managing bare metal
+hosts.  It maintains an inventory of available hosts as instances of the
+`BareMetalHost` Custom Resource Definition.  The Bare Metal Operator knows how
+to:
+
+* Inspect the host’s hardware details and report them on the corresponding
+  `BareMetalHost`.  This includes information about CPUs, RAM, disks, NICs, and
+  more.
+* Provisioning hosts with a desired image
```

Comment:
nit: to be consistent with the verb forms in the rest of this list, s/Provisioning/Provision/

Reply:
done, thanks!